FrontEnd/src/components/ProfilePage/FormComponents/GeneralInfo.js

```diff
@@ -50,6 +52,8 @@ const ERRORS = {
 const TEXT_AREA_MAX_LENGTH = 2000;
 const BANNER_IMAGE_SIZE = 5 * 1024 * 1024;
 const LOGO_IMAGE_SIZE = 1 * 1024 * 1024;
+const RNOKPP_PATTRN = /^\d{10}$/;
+const EDRPOU_PATTERN = /^\d{8}$/;
```

Comment:
These patterns are not only used in the validation functions. So that's not a good idea to move them into functions. And also they are needed only in GeneralInfo component. I don't think it makes sense to move patterns to utils.

Reply:
I see. 
if (profile.edrpou && (!EDRPOU_PATTERN.test(profile.edrpou) || !validateEdrpouKey(profile.edrpou))){
            isValid = false;
        }
Still I suggest to keep all the validation incapsulated in a function that is responsible for required element. Single Responsibility principle, although we don't use OOP here.

Reply:
Validation has been updated based on your suggestions.